recommendation/system.linq.dynamic.json

```diff
@@ -0,0 +1,73 @@
+﻿{
+  "Name": "System.Linq.Dynamic",
+  "Version": "1.0.0",
+  "Packages": [
+    {
+      "Type": "Nuget"
+    }
+  ],
+  "Recommendations": [
+    {
+      "Type": "Namespace",
+      "Name": "System.Linq.Dynamic",
+      "Value": "System.Linq.Dynamic",
+      "KeyType": "Name",
+      "ContainingType": "",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
```

Comment:
Isn't this from the community?

Reply:
If it is, the only thing we would have from the submitter would be their email address. Should be use that, since its private customer information on a public repo, or just a regular title such as "Community" ?

Reply:
This was from Amazon, so I will leave the source as Amazon unless there are differing opinions here